docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,6 +2,61 @@
 
 These workflows for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
+_Last updated 27-Mar-2024_
+### Preferred Instructions
+
+1. Open the `src/ontology/imports/manual_seed.txt`
+2. Add the IRI of the term(s) you want to add to this document, and save
+    - you can add any IRI, e.g. gene, NCBITaxon,...
+3. In the Terminal, run: `export "MEMORY_GB=15"`
+4. Then run import: `sh run.sh make refresh-merged` 
+    - All the imports will be updated, which means that you might see changes in your GitHub diff in the following files
+        - `src/ontology/imports/*_terms.txt`
+        - `src/ontology/imports/merged_import.owl`
+    - The terms added in the "manual_seed.txt" will be added to the appropriate import (e.g human genes will be added to hgnc_terms.txt; NCBITaxon will be added to ncbitaxom_terms.txt), and removed from the "manual_seed.txt" after the import refresh has run. 
+5. Edit in Protege, the new terms should be available for logical definitions.
+
+**Note when adding a new NCBITaxon**
+Check if the new NCBI Taxon identifier(s) exist in [https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt](https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt). 
+    - if the identifiers are not in the file, create a PR in this repo to include the new identifiers
+    Note: this file contains CURIEs (not IRIs) so the identifier should be added in this format `NCBITaxon:1`
+
+This additional step is needed since we are not using NCBI Taxon directly, but the OBO slim, and this file is the seed of the NCBITaxon slim.
+ 
+
+
+### Alternate instructions
```

Comment:
@matentzn can you please confirm that these alternate instructions are ok?

Reply:
Note that when I tested, running the import "sh run.sh make refresh-merged" did not bring the information for NCBITaxon. So there might be more to do?

Reply:
No, I cant think of a reason why it should not work! You will need to ask Trish for help

Reply:
yeah, there was a ticket for the one taxon that was not added but i can't find that ticket now.

Reply:
@sabrinatoro can you point me to that ticket?

Reply:
I think this is what you are thinking of: https://github.com/monarch-initiative/mondo/issues/7343

Reply:
Here it is https://github.com/monarch-initiative/mondo/issues/7453.
But I don't think it will be useful, especially since I used this updated workflow and successfully added it ([PR](https://github.com/monarch-initiative/mondo/pull/7490) is awaiting for review)

Reply:
yep, found that and reviewed/merged. 

Are the "Detailed instructions for adding..." sections still needed?